Form/JQuery/Type/Select2Type.php

```diff
@@ -89,13 +90,16 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getParent()
     {
-        return $this->widget;
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\\' . ucfirst($this->widget) . 'Type')) {
+            return 'Symfony\Component\Form\Extension\Core\Type\\' . ucfirst($this->widget) . 'Type';
+        }
+        return 'Symfony\Bridge\Doctrine\Form\Type\\' . ucfirst($this->widget) . 'Type';
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
Is it symfony 2.8 or 3.0? Could you please show a code snippet on how you're using it?